simulator/src/resim/cmd_publish.rs

```diff
@@ -95,9 +95,14 @@ impl Publish {
             );
             writeln!(out, "Package updated!").map_err(Error::IOError)?;
         } else {
+            let owner_badge = hex::decode(self.owner_badge.clone())
```

Comment:
Let's remove hex. We can use manifest string for this case.

- `resim publish <path>  resource_xxxx,123u32`
- `resim publish <path>  resource_xxxx,"abc"` (users will type `\"` to work around shell)
- `resim publish <path>  resource_xxxx,456u128`

Reply:
Similarily, need to check if `resim new-owner-badge` outputs the same syntax.

Reply:
@dhedey please go ahed with implementation